packages/vue-app/template/utils.js

```diff
@@ -683,20 +603,12 @@ export function addLifecycleHook(vm, hook, fn) {
   }
 }
 
-export function urlJoin () {
-  return [].slice
-    .call(arguments)
-    .join('/')
-    .replace(/\/+/g, '/')
-    .replace(':/', '://')
-}
+export const urlJoin = joinURL
 
-export function stripTrailingSlash (path) {
-  return path.replace(/\/+$/, '') || '/'
-}
+export const stripTrailingSlash = withoutTrailingSlash
 
 export function isSamePath (p1, p2) {
-  return stripTrailingSlash(p1) === stripTrailingSlash(p2)
+  return withoutTrailingSlash(p1) === withoutTrailingSlash(p2)
```

Comment:
We may implement this in ufo :p

Reply:
Great. I think perhaps keep this PR pending until those changes are released in `ufo`?